src/axom/primal/geometry/Polyhedron.hpp

```diff
@@ -750,14 +752,14 @@ class Polyhedron
  * \note The Octahedron is assumed to have a specific vertex order:
  * \verbatim
  *
- *            0                +y
- *            /\                    +z
- *       4 --/  \-- 5           ^  >
+ *            4                +z
+ *            /\                    +y
+ *       0 --/  \-- 2           ^  >
  *         \/    \ /            | /
  *         /      \             |/
- *       2 -------- 1           -----> +x
+ *       5 -------- 3           -----> +x
  *            \/
- *            3
+ *            1
```

Comment:
Thank you, Brian!  That's excellent.  I'm relieved this function was, after all, consistent with Octahedron class.

Kenny, to me the Octahedron at https://github.com/LLNL/axom/blob/ca7e58ff86f5f16c4c1f57f3042bd911905adb13/src/axom/primal/tests/primal_octahedron.cpp#L43-L49 would be useful as an additional diagram, not the primary diagram.  This is because that octahedron is degenerate to a triangular prism: three pairs of side walls are coplanar.  When I was doing the adaptation of the subdivision algorithm, I found that hard to wrap my head around.  ASCII art is really valuable, but it does have its limitations of orientation and resolution.  But on the other hand, the subdivision algorithm is the original and still one of the important uses of the Octahedron data structure, and it uses them to represent triangular prisms.  I guess I would say, Yes, it would be good to add that ASCII art diagram, *as long as* we can make it clear.

Reply:
Kenny, somehow your comments only show up to me on the front page ("conversation" tab).  To me, though, putting +/- xyz labels on the Octahedron is misleading.  I don't see any value in modifying Octahedron.hpp.

Reply:
Thanks @agcapps -- is the triangle prism octahedron w/ the desired labeling captured as a unit test? 
E.g. is it this one?
https://github.com/LLNL/axom/blob/ca7e58ff86f5f16c4c1f57f3042bd911905adb13/src/axom/primal/tests/primal_octahedron.cpp#L35-L41
(I think it is).

If not, can we please add it to guard against regressions?

Reply:
> E.g. is it this one?

Yes.
 I'll add a diagram for both of the cases.

Reply:
Actually, I drew this out on paper; neither Octahedron in primal_octahedron.cpp lines 35 through 49 is a triangle prism.

Reply:
> Actually, I drew this out on paper; neither Octahedron in primal_octahedron.cpp lines 35 through 49 is a triangle prism.

For my own future self-reference:

35-41 is a regular octahedron like so: https://en.wikipedia.org/wiki/Octahedron#/media/File:Octahedron.jpg

43-49 is a more wonky looking octahedron that's hard to draw in 2D

Reply:
I think 43--49 is the "dual cube octahedron" shown here:
https://en.wikipedia.org/wiki/Octahedron#/media/File:Dual_Cube-Octahedron.svg
As far as I can tell, each of its edges has length $\sqrt(2)$ when the vertices are at $(\pm1,0,0)$, $(0,\pm1,0)$, and $(0,0,\pm1)$

(I.e. the cube is centered at the origin and its faces are centered at the above vertices.)